xarray/core/dataset.py

```diff
@@ -792,13 +806,19 @@ def chunks(self):
         array.
         """
         chunks = {}
-        for v in self.variables.values():
+        for v in self.data_vars.values():
             if v.chunks is not None:
                 new_chunks = list(zip(v.dims, v.chunks))
                 if any(chunk != chunks[d] for d, chunk in new_chunks
                        if d in chunks):
                     raise ValueError('inconsistent chunks')
                 chunks.update(new_chunks)
+        if chunks:
```

Comment:
I guess this method is inconsistent with `Variable.chunks`, but it currently always returns a dict.

I would either skip this change or use something like my version.